src/views/StoryViewer.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="viewer" :style="currentFrame.style">
+    <p v-if="currentFrame.text">{{ currentFrame.text }}</p>
+  </div>
+</template>
+
+<script>
+import { mapGetters, mapActions } from 'vuex'
+export default {
+  name: 'StoryViewer',
+  computed: mapGetters(['currentFrame']),
```

Comment:
I'm a bit confused here, it uses the getter from store but doesn't use the current frame component, and it's hard to track which store is being used in components/views

is this common in vuex/redux/flux - liked state management concept/lib? or this is another trade off of using it.

Reply:
There is only one store and it is global to the application.

Reply:
So added complexity will come once we can add more than one story to the (single) store... then we'll need logic for something like "get the current frame from story X". 

I agree that we'll want to reuse a current frame component in the viewer sooner than later